sources/ru/mow/statewide.json

```diff
@@ -0,0 +1,31 @@
+{
+    "coverage": {
+        "ISO 3166": {
+            "alpha2": "RU-MOW",
+            "country": "Russian Federation",
+            "subdivision": "Moscow"
+        },
+        "country": "ru",
+        "state": "MOW"
+    },
+    "data": "https://s3.amazonaws.com/data.openaddresses.io/cache/uploads/sergiyprotsiv/736ec4/ru-msk.csv.zip",
+    "website": "http://data.mos.ru/opendata/7705031674-adresniy-reestr-zdaniy-i-soorujeniy-v-gorode-moskve",
+    "license": {
+        "url": "https://apidata.mos.ru/Home/Terms",
+        "attribution": true,
+        "share-alike": false
+        },
+    "type": "http",
+    "compression": "zip",
+    "language": "ru",
+    "conform": {
+        "number": ["Housenumber","Unit","Building"],
```

Comment:
The Unit field should be moved to "unit" instead of "number".

For Building, one nice way to do this that stays true to the source might be with the format function like so:

```
"number": {
    "function": "format",
    "fields": ["Housenumber", "Building"],
    "format": "дом $1, корпус $2"
}
```

Note that if "Building" is null in this case, format will remove the intervening text so it should just be e.g. "дом 4"